Block/ProductLabel/ProductLabel.php

```diff
@@ -84,21 +158,14 @@ public function getAttributesOfCurrentProduct()
         $attributesList = array();
```

Comment:
Why retrieve "all" attributes of the current products ? Probably "retrieve all attributes that may be used for product labels" could be enough.

Reply:
But how can i check if the attribute is exist in product labels list !?